### DISCUSSION TITLE
Issue fetch().items isn't working [Urgent]

### DISCUSSION THREAD

**Discussion by innicoder** [deta/cloud-docs#258] [Help] [answered]
![image](https://user-images.githubusercontent.com/100089381/172936865-818d996f-4cd1-482a-9ac0-eb445bedbaab.png)
I have done proper setup, then db.fetch().items it doesn't crash locally but it does when deployed.

```
        "name": "news",
        "id": "7436410f-a36a-4d14-a1f4-8c52f3757e98",
```

**Comment by abdelhai** [comment, marked as answer]
hi try adding `deta` to your requirements.txt file and run `deta deploy`

**Reply by innicoder** [reply]
You would think that this would be default but for some reason it didn't give the usual errors and this worked.

**Reply by abdelhai** [reply]
it says it at the top of the official docs https://docs.deta.sh/docs/base/sdk

![Screenshot SDK Deta Docs](https://user-images.githubusercontent.com/1752577/172947788-60e4fcbe-caea-405c-be5f-ab33dce1afaf.png)

**Reply by A223D** [reply]
@abdelhai Please change the Note to a Warning so it is more visible(similar to the warning about project key). I have spent the entire day on this issue, and still somehow missed this while reading through the docs...

**Reply by jnsougata** [reply]
There's nothing to `warn` about it tbh! A developer should be atleast notice it while going through a doc!
